crates/db_schema/src/source/community.rs

```diff
@@ -218,14 +233,18 @@ pub struct CommunityFollower {
   pub community_id: CommunityId,
   pub person_id: PersonId,
   pub published: DateTime<Utc>,
-  pub pending: bool,
+  pub state: CommunityFollowerState,
+  pub approved_by: Option<PersonId>,
```

Comment:
`approver_id` would be better.  Like how reports have a `resolver_id`.

Reply:
Done